requirements.txt

```diff
@@ -1,6 +1,6 @@
-python-telegram-bot==13
+python-telegram-bot
```

Comment:
Can you please pin the version to a major release again? Looks like the api often introduces breaking changes.